### DISCUSSION TITLE
Notes from jaxoplanet docathon

### DISCUSSION THREAD

**Discussion by dfm** [exoplanet-dev/jaxoplanet#105] [Ideas]
Some ideas for tutorials:

1. Intro to JAX
2. Intro to NumPyro (So)
3. Data & models: Overview of elements to build an orbit (Catriona) -> Multi planet systems (et al.) 
5. Simple self contained tutorials for different types of data; include a multi planet system (Lionel: RVs, astrometry, ..., & DJ) Gaia?
6. Transit orbit vs. Keplerian
7. How to combine datasets (Lionel?, DJ)
8. Units in JAX (Dan)
9. Details of core: e.g. Mathematical discussion and reimplementation of Kepler solver (ref. autodidax) (Dan)

Other things we want to do:

1. Audit and write docstrings for public API: priority is Keplerian Orbit module (Catriona)
2. Build infrastructure for tutorials (e.g. launch in colab)
3. Release a new version to PyPI

Other notes:

1. Add note about printing objects to see default units to an early tutorial

**Comment by lgrcia** [comment]
I think we would benefit from a bit of developer documentation specifying:
- Formatting conventions (including docstrings)
- how to build the docs
- our test suite (relevant to reproduce the starry comparison tests)
